### ISSUE TITLE
Rockstar Editor infinite loading

### ISSUE THREAD

**Issue by taoletsgo** [citizenfx/fivem#2767]
### What happened?

When the server IP address is changed, opening the clip in Rockstar Editor will cause an infinite loading.
The log shows that the old IP address is being requested, and the Windows Resource Monitor also shows that the connection is to the old IP address. When not in Rockstar Editor, the connection is to the new IP address.
Clearing the DNS cache does not work, because it seems that the old IP address is written to the clip file
![1](https://github.com/user-attachments/assets/7a7e978f-dd4a-4a90-9095-6f8ea2ae4788)
![2](https://github.com/user-attachments/assets/4f868197-ab2e-4603-81bd-9529994d8c6b)

### Expected result

The correct server IP address can be resolved in Rockstar Editor

### Reproduction steps

1. Start the server, then record the clip on the client side, we can load the clip
2. Stop the server and change the server IP address
3. Start the server, then open the rockstar editor on the client side, it will be stuck in infinite loading clips

### Importancy

Crash

### Area(s)

FiveM, FXServer

### Specific version(s)

fx: b9572 client:2699

**Comment by Ivek21** [comment]
+

**Comment by Ivek21** [comment]
++

**Comment by AvarianKnight** [comment]
This doesn't seem like a bug?

**Comment by taoletsgo** [comment]
> This doesn't seem like a bug?

Yeah, it doesn't seem to be a bug or a DNS cache, because each connection to the resource cache in the editor will wait more than 4 times before throwing an error and verifying the next resource cache. 

https://github.com/citizenfx/fivem/blob/master/code/components/citizen-resources-client/src/ResourceCacheDeviceV2.cpp#L594

If the server has a large number of script resources, it will cause a very long loading time. My server has 40 scripts, and usually after waiting for 10 minutes, it will trigger an early exit trap and exit.
But if the server IP has not changed, the resource cache can be verified quickly and the clip can be loaded.
